cicd-scripts/customize-mco.sh

```diff
@@ -42,6 +46,13 @@ update_mco_cr() {
     ${SED_COMMAND} "/annotations.*/a \ \ \ \ mco-imageTagSuffix: ${LATEST_SNAPSHOT}" ${ROOTDIR}/examples/mco/e2e/v1beta1/observability.yaml
     ${SED_COMMAND} "/annotations.*/a \ \ \ \ mco-imageTagSuffix: ${LATEST_SNAPSHOT}" ${ROOTDIR}/examples/mco/e2e/v1beta2/observability.yaml
 
+    # we need to change the mco operator img in Prow KinD cluster
+    if [[ -n "${IS_KIND_ENV}" ]]; then
+        cd ${ROOTDIR}/operators/multiclusterobservability/config/manager && kustomize edit set image quay.io/open-cluster-management/multicluster-observability-operator=${MULTICLUSTER_OBSERVABILITY_OPERATOR_IMAGE_REF}
```

Comment:
move it to setup-e2e-test.sh so that you do not need to make changes in `run-e2e-tests.sh`
```
if [[ -z "${IS_KIND_ENV}" ]]; then
    ./cicd-scripts/customize-mco.sh
fi
```